proto/inbound.proto

```diff
@@ -71,20 +71,27 @@ message ProxyProtocol {
     // matched.
     repeated HttpRoute http_routes = 3;
 
+    // If the protocol detected as HTTP, applicable rate limit.
+    HttpLocalRateLimit local_rate_limit = 4;
```

Comment:
Because the detect variant may handle non-HTTP protocols, let's name it unambiguously:

```suggestion
    HttpLocalRateLimit http_local_rate_limit = 4;
```

This consistent with the way that we call the routes `http_routes` here and `routes` in the HTTP variants.